### ISSUE TITLE
Training on human data

### ISSUE THREAD

**Issue by vivekmadhavaram** [caiyuanhao1998/Open-DiffusionGS#21]
Hi @caiyuanhao1998,

I am training the model on a human face dataset, and after a few training iterations, the generated results start to degrade significantly. I have attached sample outputs below that illustrate the issue I am encountering.

Initially, the model appears to learn reasonable structure, but very quickly the outputs collapse into distorted or noisy patterns. This behavior suggests a potential issue related to data preprocessing, camera parameters, coordinate conventions, normalization, or training stability (e.g., learning rate, loss scaling, or initialization).

I would appreciate your help in identifying the possible causes of this behavior and any suggestions to rectify it. Please let me know if additional details—such as training configuration, dataset format, or hyperparameters—would be helpful for debugging.

Thank you for your time and support.

<img width="3600" height="516" alt="Image" src="https://github.com/user-attachments/assets/3c29e21e-b105-4f8b-b89e-0adf000c38a8" />
<img width="3600" height="516" alt="Image" src="https://github.com/user-attachments/assets/2ea7d754-0a0e-4fce-84b9-13db4cd7c0d3" />.

**Comment by caiyuanhao1998** [comment]
Hi, thanks for your interest,

I suggest you tweak down the lr and then do a normalized gradient thresholding. 

If you find our repo useful, please help us star it, thanks :)
